extensions/amp-mustache/0.2/amp-mustache.js

```diff
@@ -133,13 +135,15 @@ export class AmpMustache extends AMP.BaseTemplate {
   }
 }
 
-// First, unregister template with same type to avoid "Duplicate template type"
-// error due to multiple versions of amp-mustache in the same unit test run.
-// This is due to transpilation of test code to ES5 which uses require() and,
-// unlike import, causes side effects (AMP.registerTemplate) to be run.
-// For unit tests, it doesn't actually matter which version of amp-mustache is
-// registered. Integration tests should only have one script version included.
-if (getMode().test) {
-  Services.templatesFor(window).unregisterTemplate('amp-mustache');
-}
-AMP.registerTemplate('amp-mustache', AmpMustache);
+AMP.extension(TAG, '0.2', function(AMP) {
+  // First, unregister template with to avoid "Duplicate template type"
```

Comment:
Remove "with" in comment.

// First, unregister template **with** to avoid "Duplicate template type. 

Also there is a test failure now that seems to indicate that moving this logic is causing this issue to happen again.

Reply:
Done.